setup.cfg

```diff
@@ -42,6 +42,7 @@ dev =
   python-dotenv==1.0.0
   ruff==0.0.292
   pytest-asyncio==0.23.5
+  openai>=1.12.0,<1.12.9
```

Comment:
We might have to allow all the 1.x.x versions here as openai is already on 1.13.0 version. Might have to test this as well by installing portkey to a project that already has openai installed in it.

Reply:
Done